### DISCUSSION TITLE
Stop extension from prompting for 2FA every time a folder or window is opened

### DISCUSSION THREAD

**Discussion by socketbox** [aws/aws-toolkit-vscode#4454] [Q&A]
Is there any way I can stop the extension from prompting for MFA each time I open a VSCode window or folder? I see only the ability (in the extension settings) of suppressing the Telemetry confirmation prompt.

Here's the pop-up I'm referring to: 
![image](https://github.com/aws/aws-toolkit-vscode/assets/1593835/ee3925a0-d5a0-4cdc-8a4f-ce3fb73a98a2)

**Comment by justinmk3** [comment]
Which 2FA prompt do you mean exactly? Can you show a screenshot? 

Are you using a AWS IAM profile with [MFA](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa.html) ? As a workaround, try selecting a different profile.
